static/js/apps/explore/follow_up_questions.tsx

```diff
@@ -52,11 +52,17 @@ interface FollowUpQuestionsPropType {
 export function FollowUpQuestions(
   props: FollowUpQuestionsPropType
 ): ReactElement {
-  const ref = useRef(null);
-  const isOnScreen = useOnScreen(ref);
-  const [hasReportedView, setReportedView] = useState(false);
   const [followUpQuestions, setFollowUpQuestions] = useState(null);
   const [loading, setLoading] = useState(true);
+  const { ref } = useInView({
```

Comment:
nit: considering renaming `ref` to something like `inViewRef` or something a little more descriptive.